presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestTimestampCompatibility.java

```diff
@@ -78,4 +86,22 @@ public void testTimestampCompatibility(String storageFormat, String legacyTimest
 
         resetSessionProperty(connection, "legacy_timestamp");
     }
+
+    @Test(dataProvider = "storage_formats", groups = {HIVE_CONNECTOR, TIMESTAMP})
+    public void testTimestampWithNullCompatibility(String storageFormat, String legacyTimestamp)
```

Comment:
do you need to have separate group of tests for that? maybe lests merge this two test methods together.